### PR TITLE
Some minor doc corrections

### PR DESCRIPTION
**Pull request by jrenaat**

### CODE REVIEW

File: documentation/src/main/asciidoc/userguide/chapters/domain/collections.adoc [hunk 1]
```diff
@@ -132,7 +132,7 @@ include::{example-dir-collection}/classification/list/EntityWithIndexBasedList.j
 
 
 
-[[collections-set]]
+[[collections-set-mapping]]
```
Comment (beikov) [inline documentation/src/main/asciidoc/userguide/chapters/domain/collections.adoc:135]:
```suggestion
[[collections-set]]
[[collections-set-mapping]]
```

File: documentation/src/main/asciidoc/userguide/chapters/domain/collections.adoc [hunk 2]
```diff
@@ -201,7 +201,7 @@ instead.
 
 
 
-[[collections-map]]
+[[collections-map-mapping]]
```
Comment (beikov) [inline documentation/src/main/asciidoc/userguide/chapters/domain/collections.adoc:204]:
```suggestion
[[collections-map]]
[[collections-map-mapping]]
```

File: documentation/src/main/asciidoc/userguide/chapters/domain/collections.adoc [hunk 3]
```diff
@@ -242,7 +242,7 @@ include::{example-dir-collection}/classification/map/EntityWithSortedMap.java[ta
 
 
 
-[[collections-bag]]
+[[collections-mapping]]
```
Comment (beikov) [inline documentation/src/main/asciidoc/userguide/chapters/domain/collections.adoc:245]:
```suggestion
[[collections-bag]]
[[collections-mapping]]
```

File: documentation/src/main/asciidoc/userguide/chapters/domain/collections.adoc [hunk 4]
```diff
@@ -293,14 +293,14 @@ table.  `@CollectionId` is the annotation to configure this identifier
 
 
 
-[[collections-array]]
+[[collections-array-mapping]]
```
Comment (beikov) [inline documentation/src/main/asciidoc/userguide/chapters/domain/collections.adoc:296]:
```suggestion
[[collections-array]]
[[collections-array-mapping]]
```

File: documentation/src/main/asciidoc/userguide/chapters/domain/embeddables.adoc [hunk 5]
```diff
@@ -611,7 +610,7 @@ create table JsonHolder as (
 
 Again, the name and the nullability of the `aggregate` column can be refined through applying a `@Column` on the persistent attribute.
 
-[[embeddable-mapping-aggregate]]
+[[embeddable-mapping-collections]]
```
Comment (beikov) [inline documentation/src/main/asciidoc/userguide/chapters/domain/embeddables.adoc:613]:
```suggestion
[[embeddable-mapping-aggregate]]
[[embeddable-mapping-collections]]
```

**Review by beikov — COMMENTED** [review]
Instead of removing anchors on which people might already rely on, I suggest you just add additional ones.

**Comment by jrenaat** [comment]
> Instead of removing anchors on which people might already rely on, I suggest you just add additional ones.

Yeah, while this is true Christian, it also leaves the docs littered with either references that are no longer valid or conflicting (case in point, repeated refs) that lead to invalid links that 'clients' of these will most likely stop using or remove anyway, because of that.
So your suggestion kinda defeats the purpose of this PR, which was to correct (some of) these problems.

**Comment by beikov** [comment]
If an anchor is wrong, I agree to remove/replace it, but let's be sensible here, links to these sections (e.g. https://docs.jboss.org/hibernate/orm/5.0/userguide/html_single/Hibernate_User_Guide.html#collections-set) have used these anchors for quite some time and are possibly linked a few times already. Let's not just break peoples links. I think we can deal with the occasional double anchor in our minds for the sake of our users.

**Pull request closed without merging**